test/e2e/multicontainer_test.go

```diff
@@ -28,14 +28,13 @@ import (
 	"knative.dev/pkg/test/spoof"
 	v1 "knative.dev/serving/pkg/apis/serving/v1"
 	"knative.dev/serving/test"
-	"knative.dev/serving/test/e2e"
 	v1test "knative.dev/serving/test/v1"
 )
 
 func TestMultiContainer(t *testing.T) {
 	t.Parallel()
 
```

Comment:
Should probably skip this if beta features aren't enabled, like we do in the conformance test for this: https://github.com/knative/serving/blob/fc6ca85e4ee6f2851028149a801ae41263038e9d/test/conformance/api/v1/service_test.go#L627.

Reply:
Sure! Added the it. Thank you.